code/modules/mob/living/carbon/alien/alien_base.dm

```diff
@@ -25,6 +25,7 @@
 /mob/living/carbon/alien/Initialize(mapload)
 	. = ..()
 	create_reagents(1000)
+	faction = list("alien")
```

Comment:
Please put this in the type declaration above

Reply:
Donezo!